fileuploader-kiwiirc-plugin/plugin.js

```diff
@@ -1,5 +1,6 @@
 import { Core as Uppy, Dashboard, Tus, Webcam } from 'uppy'
 import 'uppy/dist/uppy.min.css'
+import fileListComponent from './components/fileListComponent.vue'
```

Comment:
`Component` doesn't need to be in the filename here, as we already know its a component with the folder its inside. How about `SidebarFilelist.vue` to be descriptive?